src/main/java/com/redhat/cloud/notifications/models/BehaviorGroup.java

```diff
@@ -51,6 +51,7 @@
     private String name;
 
     @NotNull
+    @JsonProperty(value = "display_name")
```

Comment:
openapi was reporting it as "displayName", but returned as display_name

Reply:
Both `@JsonProperty` and `@Schema` can fix that, but I would favor `@Schema` here because smallrye-open-api will soon [support a global naming strategy](https://github.com/smallrye/smallrye-open-api/pull/788) and we will remove all `@Schema` annotations when that happens.

```suggestion
    @Schema(name = "display_name")
```